daemon/algod/api/server/v1/handlers/handlers.go

```diff
@@ -1615,9 +1615,7 @@ func Transactions(ctx lib.ReqContext, context echo.Context) {
 	// swagger:operation GET /v1/account/{address}/transactions Transactions
 	// ---
 	//     Summary: Get a list of confirmed transactions.
-	//     Description: Returns the list of confirmed transactions between within a date range. This call is available only when the indexer is running.
-	//     Produces:
-	//     - application/json
+	//     Description: Returns the list of confirmed transactions between within a date range. When indexer is disabled this call requires firstRound and lastRound and returns an error if firstRound is not available to the node. Transactions are returned in round, then block order.
```

Comment:
it took me few seconds to figure what you meant by " round, then block order."
( as block order is usually identical to round order )

I think you means in-block or on-block order ( I'm not sure which one is the better way to write this )